spec/spec_helper.rb

```diff
@@ -37,6 +37,10 @@ module Features
   config.include Paperclip::Shoulda::Matchers
   config.include ActionDispatch::TestProcess
 
+  config.before(:each) do
+    ENV['SINGLE_TENANT_MODE'] = 'false'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.